### PR TITLE
New sidebar and theme

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -109,14 +110,14 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
-    rack (2.1.4)
+    rack (1.6.13)
```
Comment (payex-dev) [inline Gemfile.lock:113]: Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 3
           

Complexity increasing per file
==============================
- _plugins/sidebar.rb  12
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: _plugins/sidebar.rb [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                      "<li class=\"nav-leaf nav-subgroup-leaf\"><a href=\"#{value[:url]}\">#{title}</a></li>"
+                    else
+                      "<li class=\"nav-leaf\"><a href=\"#{value[:url]}\">#{title}</a></li>"
+                    end
+      end
+
+      subgroup
+    end
+
+    def render(filename)
```
Comment (payex-dev) [inline _plugins/sidebar.rb:128]: Codacy found an issue: [Cyclomatic complexity for render is too high. [7/6]](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: _plugins/sidebar.rb [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+    private
+
+    def sanitize_filename(filename)
+      sanitized_filename = filename.match(/(?m)(?<=\b_site).*$/)[0]
+      sanitized_filename = sanitized_filename.gsub('index.html', '')
+      sanitized_filename.gsub('.html', '')
+    end
+
+    def generateSubgroup(filename, key, value, all_subgroups, level)
```
Comment (payex-dev) [inline _plugins/sidebar.rb:78]: Codacy found an issue: [Cyclomatic complexity for generateSubgroup is too high. [12/6]](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

Comment (payex-dev) [inline _plugins/sidebar.rb:78]: Codacy found an issue: [Perceived complexity for generateSubgroup is too high. [16/7]](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+
+```mermaid
+sequenceDiagram
+    participant SwedbankPay as Swedbank Pay
+    participant Merchant
+    participant Consumer
+    participant Trustly
+
+    Consumer->>Merchant: Start purchase
+    activate Merchant
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+    note left of Merchant: First API request
+    Merchant->>-SwedbankPay: POST /psp/trustly/payments
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: rel: view-authorization
+    activate Merchant
+    Merchant-->>-Consumer: Display all details and final price
+    activate Consumer
+    note left of Consumer: Open iframe ③
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:258]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+    Consumer->>Consumer: Input first name and last name
+    Consumer->>-SwedbankPay: Confirm purchase
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+    participant Consumer
+    participant Trustly
+
+    Consumer->>Merchant: Start purchase
+    activate Merchant
+    note left of Merchant: First API request
+    Merchant->>-SwedbankPay: POST /psp/trustly/payments
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: rel: view-authorization
+    activate Merchant
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+    participant Merchant
+    participant Consumer
+    participant Trustly
+
+    Consumer->>Merchant: Start purchase
+    activate Merchant
+    note left of Merchant: First API request
+    Merchant->>-SwedbankPay: POST /psp/trustly/payments
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: rel: view-authorization
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md [hunk 9]
```diff
@@ -0,0 +1,290 @@
...
+    activate Merchant
+    Merchant-->>-Consumer: Display all details and final price
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:256]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md [hunk 10]
```diff
@@ -0,0 +1,290 @@
...
+    activate Consumer
+    note left of Consumer: Open iframe ③
+    Consumer->>Consumer: Input first name and last name
+    Consumer->>-SwedbankPay: Confirm purchase
+    activate SwedbankPay
+    SwedbankPay->>-Trustly: Perform payment
```
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md (hunk 7 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 6 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 9 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:256]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 8 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 4 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 5 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:258]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md (hunk 6 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 7 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 8 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 9 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:256]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 4 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 8 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 6 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 5 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:258]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 7 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 4 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md (hunk 8 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 5 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:258]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 9 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:256]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 7 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 6 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 4 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md (hunk 6 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 4 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 7 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 8 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 5 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:258]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 9 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:256]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: payments/trustly/seamless-view.md (hunk 8 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:254]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 5 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:258]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 9 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:256]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 4 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:250]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 7 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:255]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:262]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

File: payments/trustly/seamless-view.md (hunk 6 above)
Comment (payex-dev) [inline payments/trustly/seamless-view.md:260]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908)

**Review by marinafrydenb — APPROVED** [review]
Amazing! im so exited!

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Review by asbjornu — APPROVED** [review]
Whoop! 🎉

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: .github/workflows/test.yml [hunk 11]
```diff
@@ -4,9 +4,14 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+
+      - name: Login docker
```
Comment (asbjornu) [inline .github/workflows/test.yml:11]:
```suggestion
      - name: docker login
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

File: .github/workflows/test.yml [hunk 12]
```diff
@@ -4,9 +4,14 @@ on: [push, pull_request]
...
+        run: |
+          docker login https://docker.pkg.github.com -u SwedbankPay --password ${{ secrets.GITHUB_TOKEN }}
```
Comment (asbjornu) [inline .github/workflows/test.yml:13]:
```suggestion
        run: docker login https://docker.pkg.github.com -u SwedbankPay --password ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/test.yml [hunk 13]
```diff
@@ -20,17 +25,20 @@ jobs:
...
   verify:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - name: Verify build
+
+      - name: Login docker
+        run: |
+          docker login https://docker.pkg.github.com -u SwedbankPay --password ${{ secrets.GITHUB_TOKEN }}
```
Comment (asbjornu) [inline .github/workflows/test.yml:34]:
```suggestion
      - name: docker login
        run: docker login https://docker.pkg.github.com -u SwedbankPay --password ${{ secrets.GITHUB_TOKEN }}
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002908&bid=19679799)

**Review by asbjornu — APPROVED** [review]

**Pull request merged**